configure

```diff
@@ -3,36 +3,16 @@
 ## This micro-configure shell script is here only to
 ## launch the real configuration via ocaml
 
-ocaml=ocaml
-script=./configure.ml
-
-if [ ! -f $script ]; then
-    echo "Error: file $script not found in the current directory."
-    echo "Please run the configure script from the root of the coq sources."
-    echo "Configuration script failed!"
-    exit 1
+dune=dune
+configure=./tools/configure/configure.exe
+
+## Check that dune is available, provide an error message otherwise
+# EJGA: to my shame I wasn't able to make this work on dash...
+# if ! command -v $dune &> /dev/null
```

Comment:
```suggestion
if ! command -v $dune > /dev/null
```

Reply:
Still doesn't work in dash , thanks for the suggestion :( :(

[or I am very dumb and missing something obvious]

Reply:
works for me, what does your dash say?

Reply:
It just always takes the if branch :S